pallas-traverse/src/header.rs

```diff
@@ -45,4 +46,57 @@ impl<'b> MultiEraHeader<'b> {
             MultiEraHeader::Byron(x) => x.to_hash(),
         }
     }
+
+    pub fn leader_vrf_output(&self) -> Result<ByteVec, Error> {
+        match self {
+            MultiEraHeader::EpochBoundary(_) => Err(Error::InvalidEra(0)),
+            MultiEraHeader::AlonzoCompatible(x) => Ok(x.header_body.leader_vrf.0.clone()),
+            MultiEraHeader::Babbage(x) => {
+                let mut leader_tagged_vrf: Vec<u8> = vec![0x4C_u8]; /* "L" */
+                leader_tagged_vrf.extend(&*x.header_body.vrf_result.0);
+                let h32 = Hasher::<256>::hash(&leader_tagged_vrf);
+                Ok(ByteVec::from(h32.to_vec()))
```

Comment:
I think we can skip the ByteVec wrapper and just return a Hash<32>. The bytevec is a cbor-specific decoding struct which should remain as hidden as possible from the api surface.

Reply:
Went with a simple `Vec<u8>`.